doc/changelog.md

```diff
@@ -132,6 +132,9 @@
         (array([0.5, 0. , 0.5, 0. ]),)  
   ```
   
+* Fix decomposition tests.
+  [(#745)](https://github.com/PennyLaneAI/catalyst/pull/745)
```

Comment:
I don't think this PR contains any changes to the package, as such it probably shouldn't feature a changelog entry.

Reply:
See here: https://github.com/PennyLaneAI/catalyst/pull/745/commits/c1297dd974885d5ca460e615c1e2d25de3ecddd9